Bottom: e1c8e687ff0612621def9ab3a55bfaefac34116c
Top:    7f2d3754d560f9ebe0491f8cda0ad1d5efcfa1a9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:45:33 +0000

Refresh of hid-common-provide-a-header-fi

---

diff --git a/src/hid/common/actions.h b/src/hid/common/actions.h
new file mode 100644
index 0000000..1cc6f24
--- /dev/null
+++ b/src/hid/common/actions.h
@@ -0,0 +1,7 @@
+
+#ifndef __HID_ACTIONS_INCLUDED__
+#define __HID_ACTIONS_INCLUDED__
+
+void print_actions (void);
+
+#endif
